package.json

```diff
@@ -8,7 +8,7 @@
     "lint": "eslint --ext .js",
     "server-test": "mocha -R spec --recursive",
     "browser-test": "karma start",
-    "test": "npm run server-test && npm run browser-test"
+    "test": "npm run server-test"
```

Comment:
Then it should be issue on the generator repo I guess. (replying mobile,
can't file it right away)
On 29 Dec 2015 14:12, "Almad" <notifications@github.com> wrote:

> In package.json
> <https://github.com/apiaryio/blueprint-markdown-renderer/pull/2#discussion_r48539222>
> :
>
> > @@ -8,7 +8,7 @@
> >      "lint": "eslint --ext .js",
> >      "server-test": "mocha -R spec --recursive",
> >      "browser-test": "karma start",
> > -    "test": "npm run server-test && npm run browser-test"
> > +    "test": "npm run server-test"
>
> See the comment below; this never worked, and I am totally fine adding it,
> but it's for a separate PR and development time.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apiaryio/blueprint-markdown-renderer/pull/2/files#r48539222>
> .
>